holopy/core/tests/test_metadata.py

```diff
@@ -0,0 +1,666 @@
+import unittest
+from collections import OrderedDict
+
+import numpy as np
+from nose.plugins.attrib import attr
+
+from holopy.core.metadata import (
+    detector_grid, detector_points, clean_concat, update_metadata,
+    get_spacing, get_extents, copy_metadata, make_subset_data)
+from holopy.core.errors import CoordSysError
+
+
+TOLS = {'atol': 1e-15, 'rtol': 1e-15}
+METADATA_VALUES = {
+    'medium_index': 1.234,
+    'illum_wavelen': 0.567,
+    'illum_polarization': (1, 0),
+    'noise_sd': 0.89,
+    }
+
+
+# TODO: test that:
+# update_metadata raises an error when things are not the correct shape
+#       (e.g. normals have 11 elements for a 10x10 detector)
+# FIXME questions:
```

Comment:
1. Both ```data_grid``` and ```detector_grid``` don't allow for specifying normals at each point on the detector. Is this the desired behavior? I think so, just want to confirm / hear any counter arguments why this might be desirable.

Reply:
Thanks for resolving that this is the expected behavior.